Tests/ApolloTests/Cache/StoreSubscriptionTests.swift

```diff
@@ -1,24 +1,62 @@
 import Nimble
 import XCTest
 @testable import Apollo
+//@testable import ApolloSQLite
 
-class StoreSubscriptionTests: XCTestCase {
+open class StoreSubscriptionTests: XCTestCase {
   static let defaultWaitTimeout: TimeInterval = 1
 
+  var cache: NormalizedCache!
   var store: ApolloStore!
 
-  override func setUpWithError() throws {
+  open override func setUpWithError() throws {
     try super.setUpWithError()
-    store = ApolloStore()
+    cache = InMemoryNormalizedCache()
+    //cache = try! SQLiteNormalizedCache(fileURL: URL(fileURLWithPath: "/tmp/test.sqlite"))
+    store = ApolloStore(cache: cache)
   }
 
-  override func tearDownWithError() throws {
+  open override func tearDownWithError() throws {
+    cache = nil
     store = nil
     try super.tearDownWithError()
   }
 
   // MARK: - Tests
 
+  func testUnsubscribeRemovesSubscriberFromApolloStore() throws {
+    let subscriber = NoopSubscriber()
+
+    store.subscribe(subscriber)
+
```

Comment:
I think a check in between `subscribe` and `unsubscribe` to ensure that `subscribers` is not empty at this point would be beneficial to the test.